homeassistant/components/devolo_home_control/sensor.py

```diff
@@ -180,15 +177,11 @@ def unique_id(self) -> str:
 
     def _sync(self, message: tuple) -> None:
         """Update the consumption sensor state."""
-        if message[0] == self._attr_unique_id and message[2] != "total_since":
+        if message[0] == self._attr_unique_id:
```

Comment:
This change needs to be reviewed by a code owner. I'm not sure that this change in scope is correct for the remaining cases.

Reply:
I think it's fine, this reverts changes added in https://github.com/home-assistant/core/pull/53337, which was opened by @Shutgun . Also, @2Fake and @Shutgun are the codeowners for this integration 👍

Reply:
This listened to a websocket message, if the last_rest was resetted to now. If the attribute is removed, We don't need to listen to the message anymore.

Reply:
Right, the removal of the last reset is ok. My question is about the change in scope of the remaining cases.

Reply:
True, what leads to an unneeded update of the sensor's state. As this does not make any I/O, I would be fine with that. If you have a reason, why the unneeded update should be prevented, I should cover that in a different MR to cover more message types I don't care about currently.

Reply:
Ok, thanks!